src/css/custom.css

```diff
@@ -349,3 +349,14 @@ span.callout + p::after {
 .dmn-decision-table-container .tjs-table {
   width: 100%;
 }
+
+.theme-doc-markdown img {
+  border-style: solid;
+  border-color: lightgray;
+  border-width: 1px;
+  box-shadow: 4px 4px 4px 4px lightgray;
```

Comment:
```suggestion
  box-shadow: 4px 4px 4px 1px lightgray;
```

I'm 100% certain you were working off of my code, but I recommend playing with the last numeric value of this property. I changed it to 1px, for example, and I think it gives a softer edge, and removes the alignment effect from your original value: 

<img width="961" alt="image" src="https://user-images.githubusercontent.com/1627089/214384688-23ca9452-bf38-4d5e-9263-d159f2e0e8ce.png">